Casks/optimus-player.rb

```diff
@@ -0,0 +1,28 @@
+cask "optimus-player" do
+  version "1.4"
+  sha256 "6709f0789f1bc189c38c1770da6a410a17063f431268cb808444832820b24941"
+
+  url "https://download.optimusplayer.com/Optimus%20Player%20#{version}.dmg"
+  name "Optimus Player"
+  desc "The best app for playing audiovisual content on the Mac"
```

Comment:
```suggestion
  desc "Media player"
```

The only relevant information in there is “playing audiovisual content”.